004 - Tres numeros y sumarlos.py

```diff
@@ -1,10 +1,10 @@
 #Escribir un programa que solicite al usuario ingresar tres numeros enteros.El programa
 #debe mostrar por pantalla el resultado de sumar los tres numeros de la siguiente manera: "[numero1] + [numero2] + [numero3] = [suma]".
 
-numero1 = int(input("Ingrese el primer numero: "))
-numero2 = int(input("Ingrese el segundo numero: "))
-numero3 = int(input("Ingrese el tercer numero: "))
+n1 = int(input("Numero 1: "))
+n2 = int(input("Numero 2: "))
+n3 = int(input("Numero 3: "))
 
-suma = (numero1 + numero2 + numero3)
+suma = n1 + n2 + n3
 
-print(numero1, "+", numero2, "+", numero3, "=", suma)
\ No newline at end of file
+print(f"{n1}+{n2}+{n3}= {suma}")
```

Comment:
Para que esto sea más claro, separa los símbolos y los números:
```suggestion
print(f"{n1} + {n2} + {n3} = {suma}")
```